sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -2577,8 +2577,9 @@ object SQLConf {
   val PYSPARK_JVM_STACKTRACE_ENABLED =
     buildConf("spark.sql.pyspark.jvmStacktrace.enabled")
       .doc("When true, it shows the JVM stacktrace in the user-facing PySpark exception " +
-        "together with Python stacktrace. By default, it is disabled and hides JVM stacktrace " +
-        "and shows a Python-friendly exception only.")
+        "together with Python stacktrace. By default, it is disabled to hide JVM stacktrace " +
+        "and shows a Python-friendly exception only (Note that is independent from log level" +
+        "settings. Set log level to FATAL to see Python exceptions only).")
```

Comment:
```suggestion
        "and shows a Python-friendly exception only. Note that this is independent from log "
        "level settings.")
```

Reply:
This should be enough. In production, uses won't set the log4j to log in their console in general.

Reply:
That makes sense! Thanks.